File: WarehouseManagementWeb.Application/Dto/Output/MeasureUnit/MeasureUnitOutput.cs
using WarehouseManagementWeb.Domain.Enums;

namespace WarehouseManagementWeb.Application.Dto.Output.MeasureUnit
{
    /// <summary>
    /// Класс выходной модели единицы измерения.
    /// </summary>
    public class MeasureUnitOutput
    {
        /// <summary>
        /// Id единицы измерения.
        /// </summary>
        public int Id { get; set; }

        /// <summary>
        /// Наименование единицы измерения.
        /// </summary>
        public string? Title { get; set; }

        /// <summary>
        /// Статус единицы измерения в значении перечисления.
        /// </summary>
        public DirectoryStatusEnum MeasureUnitStatusEnum { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/directory/measure-unit/models/output/measure-unit-output.ts
import { DirectoryStatusEnum } from '../../../enums/directory-status-enum';

/**
 * Класс выходной модели единицы измерения.
 */
export class MeasureUnitOutput {
  /**
   * Id единицы измерения.
   */
  id: number = 0;

  /**
   * Наименование единицы измерения.
   */
  title: string = '';

  /**
   * Статус единицы измерения в значении перечисления.
   */
  measureUnitStatusEnum: DirectoryStatusEnum = 0;
}
